turbolift/methods/archive.py

```diff
@@ -35,8 +35,14 @@ def start(self):
             self._put_container()
 
         LOG.info('Uploading Archive...')
-        with indicator.Spinner(**self.indicator_options):
-            self._upload(**archive)
+        upload_objects = self._return_deque()
+        archive_item = self._encapsulate_object(
+            full_path=archive['local_object'],
+            split_path=os.path.dirname(archive['local_object'])
+        )
+        upload_objects.append(archive_item)
```

Comment:
It might make sense to move this inside the `self._compressor` method - similar to how the `self._index_fs` returns an encapsulated object. This is required for large objects because we need the `X-Object-Manifest` header provided by `self._encapsulate_object`.